mne/viz/tests/test_montage.py

```diff
@@ -22,7 +22,8 @@
 
 
 def test_plot_montage():
-    """Test plotting montages."""
+    """Test plotting montages.
+    """
```

Comment:
This is going the wrong direction -- single-line docstrings should end with `."""` according to pydocstyle